features/yield/FarmingCalculator.tsx

```diff
@@ -271,8 +272,19 @@ const FarmingCalculator = () => {
 
   // Update APR
   useEffect(() => {
-    calculateApr();
-  }, [tokenAddress, poolLiquidity, rewardToken, rewardTokenPrices]);
+    calculateApr()
+      .then(() => setRewardPriceError(null))
+      .catch((err) => {
+        console.error(err);
+        setRewardPriceError(err);
```

Comment:
allows us to display an error if this calculator fails, which it does due to coingecko price fetch failing